app/src/main/java/com/github/dedis/student20_pop/ui/event/MeetingEventCreationFragment.java

```diff
@@ -0,0 +1,126 @@
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Locale;
+
+public final class MeetingEventCreationFragment extends AbstractEventCreationFragment {
+
+    public static final String TAG = MeetingEventCreationFragment.class.getSimpleName();
+    public static final DateFormat TIME_FORMAT = new SimpleDateFormat("HH:mm", Locale.FRENCH);
+    public static final DateFormat DATE_FORMAT = new SimpleDateFormat("dd/MM/yyyy", Locale.FRENCH);
+
+    private EditText meetingTitleEditText;
+    private EditText meetingLocationEditText;
+    private EditText meetingDescriptionEditText;
+
+    private Button confirmButton;
+    private Button cancelButton;
+
+    private OnEventCreatedListener eventCreatedListener;
+
+    @Override
+    public void onAttach(Context context) {
+        super.onAttach(context);
+        try {
+            eventCreatedListener = (OnEventCreatedListener) context;
+        } catch (ClassCastException e) {
+            throw new ClassCastException(context.toString() + " must implement OnEventCreatedListener");
+        }
+    }
+
+    @Nullable
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
+        final FragmentManager fragmentManager = (getActivity()).getSupportFragmentManager();
+        View view = inflater.inflate(R.layout.fragment_meeting_event, container, false);
+
+        setDateAndTimeView(view, MeetingEventCreationFragment.this, fragmentManager);
+        addDateAndTimeListener(confirmTextWatcher);
+
+
+        meetingTitleEditText = view.findViewById(R.id.title_text);
+        meetingLocationEditText = view.findViewById(R.id.meeting_event_location_text);
+        meetingDescriptionEditText = view.findViewById(R.id.meeting_event_description_text);
+
+        meetingTitleEditText.addTextChangedListener(confirmTextWatcher);
+
+        confirmButton = view.findViewById(R.id.confirm);
+        confirmButton.setOnClickListener(v -> {
+
+            Event meetingEvent = new MeetingEvent(
+                    meetingTitleEditText.getText().toString(),
+                    startDate,
+                    endDate,
+                    startTime,
+                    endTime,
+                    "",
+                    meetingLocationEditText.getText().toString(),
+                    meetingDescriptionEditText.getText().toString());
+
+            eventCreatedListener.OnEventCreatedListener(meetingEvent);
+
+            fragmentManager.popBackStackImmediate();
+        });
+
+        cancelButton = view.findViewById(R.id.cancel);
+        cancelButton.setOnClickListener(v -> {
+            fragmentManager.popBackStackImmediate();
+        });
+
+        // formatting today's date
+        try {
+            today = DATE_FORMAT.parse(DATE_FORMAT.format(Calendar.getInstance().getTime()));
+        } catch (ParseException e) {
+            e.printStackTrace();
```

Comment:
Maybe set a default value in case of an error

Reply:
Done